src/app/components/champion/default.ts

```diff
@@ -0,0 +1,59 @@
+export const defaultChampion = {
```

Comment:
Maybe use a spinner instead of using a default champion

Reply:
I would rather have the user directly access visual content as he loads the page, so a default champion is something I prefer (at least for now).